lib/rummager_finder_publisher.rb

```diff
@@ -31,6 +31,9 @@ def pre_production?(schema)
 
   def export_finder(schema)
     presenter = FinderRummagerPresenter.new(schema[:file], schema[:timestamp])
-    RummagerWorker.perform_async(presenter.type, presenter.id, presenter.to_json)
+
+    logger.info("Publishing '#{schema[:file]['name']}' finder")
+
+    Services.rummager.add_document(presenter.type, presenter.id, presenter.to_json)
```

Comment:
Yup you're right, this is a synchronous call rather than using the asynchronous worker. However, the reason I'm doing this is that:

1. This is only called manually as a rake task, most likely during a deployment, so it won't affect any publishing workflow
2. There are only a small number of finders and this call is pretty quick
3. It makes this publisher consistent with the publishing-api version, which also uses asynchronous calls
4. It means that changes are pretty much immediate during deployment